build/shared/examples/04.Communication/SerialPassthrough/SerialPassthrough.ino

```diff
@@ -22,12 +22,24 @@
   by Erik Nyquist
 */
 
+uint32_t currentBaud = 9600;
+
 void setup() {
-  Serial.begin(9600);
```

Comment:
begin is unnecessary for USB CDC Serial

Reply:
Is it really? I wouldn't count on this being unneccesary for *all* platforms. Also, removing the Serial.begin would remove support for serial passthrough on non-CDC boards? Typically, CDC boards ignore the baudrate passed to begin, so just calling it with 9600 should work?